src/backend/tfjs_backend.ts

```diff
@@ -409,51 +344,6 @@ export function tile(x: Tensor, n: number|number[]): Tensor {
   return tfc.tile(x, n);
 }
 
-/* Creation and manipulation of tensors and variables */
-
-/**
- * Create a Tensor with the same content as the input.
- *
- * @param x Input.
- * @return Identity output Tensor.
- */
-export function identity(x: Tensor): Tensor {
-  return x.clone();
-}
-
-/**
- * Instantiate an identity matrix and returns it, as a Variable
- *
- * @param size Number of rows/columns.
- * @param dtype Data type of returned Variable.
- * @param name Name of returned Variable.
- * @return A Variable, an identity matrix.
- */
-export function eyeVariable(
-    size: number, dtype?: DataType, name?: string): LayerVariable {
-  return new LayerVariable(tfc.eye(size, size, null, dtype), dtype, name);
-}
-
-/**
- * Multiply a scalar with an Tensor.
- * @param c The Scalar.
- * @param x The Tensor.
- * @returns The result of the multiplication.
- */
-export function scalarTimesArray(c: Scalar, x: Tensor): Tensor {
```

Comment:
In discussion before on the backend cleanup, I thought out consensus was we wanted to keep the stricter type-checks of (Scalar,Tensor)->Tensor and not replace with the less type safe raw backend call.  Same decision applied to square() as well as both scalar(Times|Plus)Array